app/TFLSettingsUI.fl

```diff
@@ -795,11 +801,11 @@ e = fl_input("Please enter a name for your PCCard Image");
 if (!e) goto cleanup;
 imageName = strdup(e);
 
-strlcat(path, imageName, sizeof(path));
+strncat(path, imageName, sizeof(path)-1);
```

Comment:
strncat/strncpy instead of strlcat/strlcpy requires checking for return values and handling of truncation. Also, we can drop dependency on libbsd.